pkg/routeagent_driver/handlers/ovn/vsctl/vsctl.go

```diff
@@ -70,6 +76,12 @@ func AddInternalPort(bridgeName, portName, macAddress string, mtu int) error {
 	return err
 }
 
+func DelInternalPort(bridgeName, portName string) error {
+	_, err := vsctlCmd("--may-exist", "del-port", bridgeName, portName)
```

Comment:
```suggestion
	_, err := vsctlCmd("--if-exists", "del-port", bridgeName, portName)
```

We should use "if-exists" during delete port.

```
Without  --if-exists,  attempting to delete a port that does not exist is an error.  With --if-exists,  attempting  to  delete  a port that does not exist has no effect.
```

Reply:
Done.